css/style.css

```diff
@@ -0,0 +1,93 @@
+#gb {
+  padding-left: 60px; /* make room for mac traffic lights */
+  -webkit-app-region: drag; /* make titlebar draggable */
+}
+
+#gbwa, /* google apps */
+#gbwa + div /* google notifications */
+ {
+  display: none;
+}
+
+#gb .gb_gd {
+  /* Compact header */
+  padding: 4px;
+}
+
+/* Remove gmail logo */
+.gb_ja:not(.gb_ka) .gb_qc .gb_Rb {
+  display: none !important;
+}
+
+/* Header */
+.gb_lc {
+  height: 32px !important;
+}
+.gb_Xb {
+  padding: 2px !important;
+  margin-left: 18px !important;
+}
+
+/* Search box */
+.gb_qc {
+  min-width: 192px !important;
+}
+
+.gb_wd.gb_Fd .gb_td .gb_ne {
+  margin-top: 10px;
+}
+
+.gb_ne,
+.gb_2e,
+.gb_Ie {
+  height: 28px !important;
+}
+
+.gb_ne button svg,
+.gb_ne button img {
+  padding: 2px !important;
+  margin: 0 !important;
+}
+
+.gb_Ie {
+  font: normal 14px Roboto, sans-serif !important;
+}
+
+/* Account avatar */
+.gb_fa > .gb_Ea {
+  padding: 0 !important;
+}
+.gb_fa {
+  border: 0px none !important;
+}
+.gb_ga {
+  display: none !important;
+}
+.gb_nc {
+  text-align: right !important;
+  width: 48px !important;
+}
+.gb_fa.gb_7f.gb_f {
+  width: 48px !important;
+}
+
+.gb_ja:not(.gb_ka) .gb_fa {
+  width: auto !important;
+  margin-left: 0px !important;
+}
+.gb_fa:hover {
+  border: 0x none !important;
+  background-color: transparent !important;
+}
+.gb_fa:focus {
+  border: 0px none !important;
+  background-color: transparent !important;
+  box-shadow: none !important;
+  -webkit-box-shadow: none !important;
+}
+
+/* Main area */
+.G-atb::before {
+  box-shadow: none !important;
+  -webkit-box-shadow: none !important;
+}
```

Comment:
I didn't really notice any difference with and without this CSS. Where exactly is this changing?

Reply:
this line under the "checkbox"
![immagine](https://user-images.githubusercontent.com/672932/55633115-ed1b4900-57bb-11e9-8350-cf2bdd90690a.png)

more "polish" interface

![immagine](https://user-images.githubusercontent.com/672932/55633178-0a501780-57bc-11e9-9cf6-0ccc2fe7a162.png)

Reply:
I want to remove also the line between every single message, but it's really my thing, for now I focus on more important things and then maybe we move on to the dark.

Reply:
Can we keep this line please? 

I think your interface looks a little different than mine:
![image](https://user-images.githubusercontent.com/8075074/55633599-a5dd8a00-57b4-11e9-84cc-7dc1baf76cd6.png)

But anyway I'd like to keep the original interface except customizing it for native looks.

Reply:
ok 👍